src/prof.c

```diff
@@ -128,7 +128,7 @@ prof_strncpy(char *UNUSED dest, const char *UNUSED src, size_t UNUSED size) {
 #endif
 }
 
-static bool
+bool
 prof_tctx_should_destroy(tsdn_t *tsdn, prof_tctx_t *tctx) {
```

Comment:
Let's switch the tsdn to tsd as well.

Reply:
Makes sense. Also added more asserts using this function to verify that the recorded `prof_tctx`s should not be destroyed (which also helped fix a bug I had).